### PR TITLE
add logInfo color option

### PR DESCRIPTION
**Pull request by ghedamat**
see https://github.com/ember-cli/ember-cli-deploy/pull/377/files

this PR implements the same changes done in the ECD repo

### CODE REVIEW

**Comment by ghedamat** [comment]
notes, this adds a dependency on a new ember-cli-deploy api (that atm is not explicit)

**Comment by lukemelia** [comment]
@ghedamat the change is backwards compatible with ember-cli-deploy versions prior to the one with the new API, confirm?

**Comment by ghedamat** [comment]
@lukemelia nope it's not. that's what I meant

now that I think more about it it should even if it will log in a bad way.. let me update and ping you

File: index.js [hunk 1]
```diff
@@ -55,15 +55,17 @@ var DeployPluginBase = CoreObject.extend({
   },
 
   log: function(message, opts) {
-    opts = opts || { color: 'blue' };
-    opts.color = opts.color || 'blue';
+    opts = opts || {};
     var ui = this.ui;
+    var color = opts.color || ui.logInfoColor || 'blue';
```
Comment (ghedamat) [inline index.js:60]: @lukemelia ok this should do (added `blue`)

**Comment by lukemelia** [comment]
:+1:

**Pull request merged**